app/controllers/admin/two_factor_authentications_controller.rb

```diff
@@ -9,6 +9,7 @@ def destroy
       @user.disable_two_factor!
       log_action :disable_2fa, @user
       UserMailer.two_factor_disabled(@user).deliver_later!
+      UserMailer.webauthn_disabled(@user).deliver_later!
```

Comment:
Yep, you're right 😅 

I think we'll be good if we just remove the email that notifies that WebAuthn was disabled and leave just the email that states that 2FA is disabled, as the first one is reductant since the latter email includes it.